frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx

```diff
@@ -28,9 +28,16 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
   return (
     <Tr>
       <Td dataLabel="Name">
-        <ResourceNameTooltip resource={project}>
-          <ProjectLink project={project} />
-        </ResourceNameTooltip>
+        <Flex spaceItems={{ default: 'spaceItemsXs' }} alignItems={{ default: 'alignItemsCenter' }}>
+          {project.metadata.labels?.[KnownLabels.DASHBOARD_RESOURCE] && (
+            <Label isCompact color="green">
+              DS
```

Comment:
We might want a tooltip saying this is `Data Science` -- `DS` is well known by us, but will it be by the users? cc @kywalker-rh (I don't know Kun's gh handle)

Reply:
cc @xianli123 👋

Reply:
@andrewballantyne That makes sense. Adding a dedicator is a good way.